src/main/webapp/app/config/constants/regex.ts

```diff
@@ -3,6 +3,6 @@
 //     (PMID: 11753428, 16007150, 21467160)
 //     (NCT1234567)
 export const REF_CAPTURE = /(\(\s*(?:PMID|NCT|Abstract):?.*?(?:\([^()]*\).*?)*\))/i;
-export const VALID_LATIN_TEXT = /^[\p{Script=Latin}\p{M}\p{N}\p{Z}\p{P}\p{Sm}\p{Sc}\p{Sk}]+$/u;
+export const VALID_LATIN_TEXT = /^[\p{Script=Latin}\p{M}\p{N}\p{Z}\p{P}\p{Sm}\p{Sc}\p{Sk}\r\n]+$/u;
```

Comment:
Do you think you should add a unit test for this?

Reply:
I do have tests, let me update them!

Reply:
@jfkonecn updated